app-nuxt-starter/nuxt.config.ts

```diff
@@ -4,5 +4,7 @@ import { createResolver } from '@nuxt/kit'
 const { resolve } = createResolver(import.meta.url)
 
 export default defineNuxtConfig({
+  // extends: ['../app-nuxt-base'],
```

Comment:
When we develop this starter locally, we need to extend it from `../app-nuxt-base` to read the local version. However, when it's used by users, it should be `github:storyblok/custom-app-examples/app-nuxt-base`. However I couldn't make it conditional, so left this comment as future usage for us.